api/v1alpha1/virtualmachineclass_types.go

```diff
@@ -118,6 +122,7 @@ type VirtualMachineClass struct {
 
 	Spec   VirtualMachineClassSpec   `json:"spec,omitempty"`
 	Status VirtualMachineClassStatus `json:"status,omitempty"`
+	Description VirtualMachineClassDescription `json:"description,omitempty"`
```

Comment:
The CRDs root shouldn't include anything else besides the `spec` and [status](url). I think the Description is better suited to be in the Spec (since it's meant to be set by the VirtualMachineClass creator)